osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -171,13 +180,41 @@ def update_order_status(self, extraction_order):
             if progress == 'successful':
                 self._download_result_files(extraction_order, job_status)
 
+    def get_country_list(self):
+        def fetch_countries():
+            self.login()
+            response = self.authorized_get(self.country_base_url)
+            if self.errors:
+                logger.error('could not fetch country list: ', self.errors)
+                return self.errors
```

Comment:
Oops, disregard that. Didn't notice `get_country_list` has a nested function in it, which this `return` applies to.

Can we un-nest those functions, though. A private function is less unexpected than a nested one. (Also, a nested one would get re-defined for each call of the outer one and garbage collected when the outer one returns.)